examples/nuxt-openai/nuxt.config.ts

```diff
@@ -2,6 +2,9 @@
 export default defineNuxtConfig({
   devtools: { enabled: true },
   modules: ['@nuxtjs/tailwindcss'],
+  nitro: {
+    preset: 'vercel-edge' // you can use 'vercel' or other providers here
+  },
   runtimeConfig: {
     openaiApiKey: ''
```

Comment:
Just a suggestion idea came up with to reduce mistakes https://github.com/pi0/nuxt-openai-demo/blob/main/nuxt.config.ts#L12